processor/src/services/converters/helper.converter.ts

```diff
@@ -28,26 +29,41 @@ export const populateLineItems = (cart: Cart): LineItem[] => {
 
   cart.customLineItems.forEach((customLineItem) => {
     lineItems.push({
+      id: customLineItem.id,
       description: Object.values(customLineItem.name)[0], //TODO: get proper locale
       quantity: customLineItem.quantity,
       amountExcludingTax: getAmountExcludingTax(customLineItem),
       amountIncludingTax: getAmountIncludingTax(customLineItem),
       taxAmount: getTaxAmount(customLineItem),
       taxPercentage: convertTaxPercentageToCentAmount(customLineItem.taxRate?.amount),
     });
-
-    if (cart.shippingInfo) {
-      lineItems.push({
-        description: 'Shipping',
-        quantity: 1,
-        amountExcludingTax: cart.shippingInfo.taxedPrice?.totalNet.centAmount || 0,
-        amountIncludingTax: cart.shippingInfo.taxedPrice?.totalGross.centAmount || 0,
-        taxAmount: cart.shippingInfo.taxedPrice?.totalTax?.centAmount || 0,
-        taxPercentage: convertTaxPercentageToCentAmount(cart.shippingInfo?.taxRate?.amount),
-      });
-    }
```

Comment:
This was incorrectly placed inside of the `cart.customLineItems.forEach` loop callback, thus it was only executed if there are any custom line items present on the cart.